src/test/resources/mockdata/invoices/invoice.json

```diff
@@ -2,6 +2,14 @@
   "id": "c0d08448-347b-418a-8c2f-5fb50248d67e",
   "approvedBy": "ab18897b-0e40-4f31-896b-9c9adc979a88",
   "approvalDate": "2018-07-29T00:00:00.000+0000",
+  "adjustments": [
+    {
+      "description": "Extra Taxes",
+      "type": "Amount",
+      "value": 10
```

Comment:
shouldn't this have `"prorated": "Not prorated"`

Reply:
correct. Added the relationToTotal as well